yellow-paper/docs/todo.md

```diff
@@ -0,0 +1,55 @@
+TODO:
```

Comment:
Whoops. Note: this doesn't actually get rendered into the book, because it's not in sidebar.js.